### ISSUE TITLE
Windows Application (menu) key doesn't work in items pane

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#1260]
https://forums.zotero.org/discussion/64508/5-0-beta-context-menu-key-on-windows-does-not-work-anymore
